com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderCompilers.java

```diff
@@ -115,23 +132,32 @@ public ArrayList<ShaderProgramBuilder.ShaderBuildResult> compile(String shaderSo
     public static class IOSShaderCompiler implements IShaderCompiler {
         public ArrayList<ShaderProgramBuilder.ShaderBuildResult> compile(String shaderSource, ES2ToES3Converter.ShaderType shaderType, String resourceOutputPath, String resourceOutput, boolean isDebug, boolean outputSpirv, boolean softFail) throws IOException, CompileExceptionError {
             ArrayList<ShaderDesc.Language> shaderLanguages = new ArrayList<ShaderDesc.Language>();
-            shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM300);
-            if (outputSpirv)
-            {
-                shaderLanguages.add(ShaderDesc.Language.LANGUAGE_SPIRV);
+
+            if (shaderType != ES2ToES3Converter.ShaderType.COMPUTE_SHADER) {
+                shaderLanguages.add(ShaderDesc.Language.LANGUAGE_GLES_SM300);
+
+                if (outputSpirv)
+                {
+                    shaderLanguages.add(ShaderDesc.Language.LANGUAGE_SPIRV);
+                }
             }
+
```

Comment:
For now, let's skip compute on mobile.

Reply:
Will they get a build error?
I guess they should, to make it clear their shader won't run?